### PR TITLE
App Insights readme and sample

### PR DESCRIPTION
**Pull request by alexeldeib**

### CODE REVIEW

**Comment by jianghaolu** [comment]
Please also add a pom file for the sample. You should be able to run `mvn clean install -DskipTests=true -B -V` first in applicationinsights/data-plane and then applicationinsights/data-plane/samples.

**Comment by jianghaolu** [comment]
@azuresdkci add to whitelist

**Comment by jianghaolu** [comment]
build java artifacts

File: applicationinsights/data-plane/samples/src/main/java/com/microsoft/azure/applicationinsights/query/samples/ApplicationInsightsClientExample.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package com.demo.app;
```
Comment (jianghaolu) [inline applicationinsights/data-plane/samples/src/main/java/com/microsoft/azure/applicationinsights/query/samples/ApplicationInsightsClientExample.java]: package is incorrect - should match folder structure `com.microsoft.azure.applicationinsights.query.samples`.

Comment (jianghaolu) [inline applicationinsights/data-plane/samples/src/main/java/com/microsoft/azure/applicationinsights/query/samples/ApplicationInsightsClientExample.java]: Add license header at the beginning.

Comment (alexeldeib) [reply applicationinsights/data-plane/samples/src/main/java/com/microsoft/azure/applicationinsights/query/samples/ApplicationInsightsClientExample.java]: fixed

Comment (alexeldeib) [reply applicationinsights/data-plane/samples/src/main/java/com/microsoft/azure/applicationinsights/query/samples/ApplicationInsightsClientExample.java]: fixed

**Comment by alexeldeib** [comment]
My lack of familiarity with Java/maven is definitely showing 😟 I can `mvn install` fine. I'm trying to execute and test the JAR maven spits out, but seems like either there's a dependency issue or this isn't a properly executable JAR? What's the easiest (or best) way to actually run this? I'd been using maven + exec plugin when I was originally testing. Seems like a very basic issue on my end. Appreciate the hand-holding you've had to do to help get this shipped!

**Comment by jianghaolu** [comment]
@alexeldeib BTW don't forget to delete the samples/ folder under src/main/java.

**Comment by jianghaolu** [comment]
Oh my bad - You already deleted from the last PR

**Comment by jianghaolu** [comment]
You are right the jars are not executable. They are library JARs.

If you import, however, both projects into IntelliJ, you will see a green "Run" button next to the Sample class definition. You can run the sample from there.

**Pull request merged**